src/handedness.rs

```diff
@@ -1,32 +1,44 @@
 use std::fmt::Debug;
 
-pub trait Handedness: Clone + Copy + Debug {
-    fn z_sign() -> f32;
-    fn right_handed() -> bool;
+use glam::Vec3;
+
+pub trait Handedness: Clone + Copy + Debug + 'static {
+    const FORWARD_Z_SIGN: f32;
+
+    fn forward() -> Vec3 {
+        Vec3::Z * Self::FORWARD_Z_SIGN
+    }
```

Comment:
I wonder if this could be made a `const fn`... Perhaps only when constructing the vector manually (ie. no multiplications because none of the builtin `*Ops` are `const fn`) using https://docs.rs/glam/latest/glam/macro.const_vec3.html.

Reply:
Making it a `const FORWARD` by using that `const_vec3` macro sounds pretty good 😀

Reply:
Looks like that's not possible? 🤔 ([E0379](https://doc.rust-lang.org/error-index.html#E0379))